<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d2_MainWindow</class>
 <widget class="QMainWindow" name="d2_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>1615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="d2_centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="d2_centerLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="d2_backgroundWidget" native="true">
        <property name="styleSheet">
         <string notr="true">#d2_backgroundWidget{
    border-image: url(:/desing2/background/master-backgrounds.png);
}


</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,18,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="d2_lblCallInfo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>58</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#d2_lblCallInfo {
	border-image: url(:/desing2/background/call-background.png)	0 0 0 0 stretch stretch;
    color : white;
}</string>
             </property>
             <property name="text">
              <string>По любым вопросам звоните: 8 800-000-00 (бесплатно 24/7)</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="d2_lblMoneyInfo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>PT Sans</family>
               <pointsize>28</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#d2_lblMoneyInfo {
	border-image: url(:/desing2/background/money-background.png) ;
    color : white;
}</string>
             </property>
             <property name="text">
              <string>Внесено: 0 Руб</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
